invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt

```diff
@@ -0,0 +1,2 @@
+-e git+git://github.com/inveniosoftware/invenio-marc21.git#egg=invenio-marc21
+-e git+git://github.com/hachreak/invenio-indexer.git@cli_reindex#egg=invenio-indexer
```

Comment:
They are temporary workaround to be able to create the `invenio` demosite. 
After the PR'll be merged, I'll restore.